pyproject.toml

```diff
@@ -44,7 +44,7 @@ dependencies = [
   'pyyaml~=6.0',
   'requests~=2.0',
   'sqlalchemy~=2.0',
-  'tabulate~=0.8.5',
+  'tabulate~=0.9.0',
```

Comment:
Wouldn't it be better for the ecosystem  to allow both 0.8 and 0.9?

Reply:
Yes it would. For packages that have not seen a v1 release, we have maintained this policy though since minor versions are likely to break things and this will break our release. I guess we could change the specifier to `tabulate>=0.8.0,<0.10.0`. It is a bit ridiculous that `tabulate` doesn't have a v1 after more than 10 years of being public...